src/uct/api/uct.h

```diff
@@ -1457,6 +1459,14 @@ typedef struct uct_md_mem_attr {
      * to uct_md_mem_query is returned as is.
      */
     size_t            alloc_length;
+
```

Comment:
minor: just noticed that `@brief ` section on line 1422 is not correct. Maybe worth fixing it here, since you update the struct anyway?